Gemfile

```diff
@@ -2,14 +2,17 @@ source "https://rubygems.org"
 
 gemspec
 
+# for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
```

Comment:
Yeah, hammer runs inside of the ``tfm`` SCL which is ruby 1.9.3.

```
# scl enable tfm "ruby --version"
ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
```

But, I guess if hammer-cli-katello still requires it then it's fine.

Question: When generating translation files, does the rake task only run inside of hammer-cli-csv? Or does a gettext require an entire loaded hammer environment with all modules?